### ISSUE TITLE
Logic error in verifyLogin function in Monitor.go (Low Priority)

### ISSUE THREAD

**Issue by BigOokie** [skycoin/skywire-testnet#100]
The `verifyLogin` function within  `Monitor.go` appears to have a logic error whereby a session value is read into the variable `hash` however the `pass` variable is checked determine if it is `nil`. Suspect this should be a check of the `hash` variable instead.  

Sections of the associated code below - the variable `pass` is checked twice.

```
verifyLogin(w http.ResponseWriter, r *http.Request, checkDefaultPass bool) bool {
	... 
	pass := sess.Get("user")
	if pass == nil {...}

	hash := sess.Get("pass")
	if pass == nil {...}
	...	
}
```

**Comment by BigOokie** [comment]
Also note that the Manager Web UI appears to enforce its own length restrictions on the user password (must be between 4 and 20 characters)

**Comment by gz-c** [comment]
Fixed by #163
